src/Extension/Extension.php

```diff
@@ -119,7 +119,16 @@ public function extend(Container $app)
             return;
         }
 
-        $extenders = array_flatten((array) require $bootstrapper);
+        $this->runExtenders($app, require $bootstrapper);
```

Comment:
I'm confused. The previous code already cast the required file into an array, so both a single item and array would become an array. There should be no need to create this unnecessary logic with the `is_array` further down the class, or am I missing something here?

Have you tried using a single Extender with the old code?

Reply:
Yes, `(array)` casts an object into an array rather than wrapping an array around the object, so the `is_array` check is necessary.